source/MaterialXGraphEditor/Main.cpp

```diff
@@ -253,7 +253,9 @@ int main(int argc, char* const argv[])
         ImGui_ImplGlfw_NewFrame();
         ImGui::NewFrame();
 
-        graph->getRenderer()->drawContents();
+        auto renderer = graph->getRenderer();
+        renderer->setFrame((renderer->getFrame() + 1) % 32768);
```

Comment:
Instead of this hardcoded maximum, perhaps we should use `std::numeric_limits<unsigned int>::max() - 1`?  That would help to clarify the motivation for the maximum, as well as providing more headroom than the existing `32768`.

Reply:
Using modulo with the counter type's maximum.